### DISCUSSION TITLE
关于自启动

### DISCUSSION THREAD

**Discussion by bzmshang** [localsend/localsend#243] [L-简体中文]
在win11上
因为微软商店不让上架后
loaclsend想要自启动
就只能放个快捷键在win的自启动目录里面
这就导致了
每次开机都得关一次loaclsend被打开的界面
而不能像在微软商店一样
直接托盘自启动
这个有没有什么办法可以解决

**Comment by wayner6** [comment]
同样的问题，请求解决

**Comment by Tienisto** [comment]
Add `autostart` parameter. See #71

**Reply by mercury321** [reply]
能否自启动后自动最小化到系统栏
